.ado/publish.yml

```diff
@@ -178,7 +208,33 @@ jobs:
       rm target/wheels/*-linux_x86_64.whl
       ls target/wheels
     displayName: Run auditwheel for Linux Wheels
-    condition: eq(variables['Agent.OS'], 'Linux')
+    condition: and(eq(variables['Agent.OS'], 'Linux'), eq(variables['arch'], 'x86_64'))
+
+  # for linux aarch64 cross build we want to skip tests as we can't run the code.
+  # and we can't run the samples as qsc is currently cross compiled.
+  - script: |
+      env CARGO_BUILD_TARGET=aarch64-unknown-linux-gnu python build.py --pip --no-check-prereqs --no-test --no-samples
```

Comment:
Do you want to do `--no-check` as well here like you did for the Windows one below?